### ISSUE TITLE
Feature Request: Calendar highlight special days

### ISSUE THREAD

**Issue by Natalia-Hristova** [telerik/nativescript-ui-feedback#77]
From @deneshr

Currently we are unable to highlight special days in the calendar such as public holidays.
Is there any way we could style specific days with a different color perhaps?

**Comment by shawndewet** [comment]
Yes please this will be a fantastic feature.  In my app, users need to indicate which days on which they are available.  I am currently doing this by means of all-day events, but this is not idea.  It would be preferable that the day itself can be indicated in a different color based on the user's availability.

**Comment by cfoulkes** [comment]
Has there been any progress on this request? Is it planned?
Thank you.
